src/get-started/codelab.md

```diff
@@ -357,6 +371,54 @@ Process finished with exit code 0
       {% include docs/ios-windows-figure-pair.md image="step2.png" alt="App at completion of second step" %}
     {% endindent %}
 
+    **You can also try out the app with a screen reader. All you need to do is turn on the screen reader on your device by following the steps below:** 
```

Comment:
**issues:** 
- Replace "below" as it runs counter to [Google's documentation accessibility guidelines](https://developers.google.com/style/accessibility).
- Use [semantic line breaks](https://sembr.org/) for copy.

```suggestion
    **You can try the app with a screen reader.
    Turn on the screen reader on your device using the following steps:** 
```